### ISSUE TITLE
input[type="url"] is not supported

### ISSUE THREAD

**Issue by Vinnl** [inrupt/solid-style-guide#59]
`<input type="url">` should probably be styled the same as `<input type="text">`.

**Comment by megoth** [comment]
It's a [supported input type though](https://caniuse.com/#feat=input-email-tel-url), so for semantics sake I think we should use it. But we usually cannot rely on the validations as that are browser specifics.

**Comment by megoth** [comment]
It's also listed as [available type on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).

**Comment by Vinnl** [comment]
Yes, I mostly meant that it's currently unstyled. It's useful even beyond validations - e.g. keyboards on mobile phones can adapt to it (with e.g. a `.com` button).
